File: test/fractures/java/Substract.java
public class Substract {
  public static void main(String[] args) {
    if (args.length < 2) {
      System.out.println("Invalid argument");
      System.out.println("Usage: java Substract <num1> <num2>");
    } else {
      int n1 = Integer.parseInt(args[0]);
      int n2 = Integer.parseInt(args[1]);
      System.out.println(n1 - n2);
    }
  }
}

File: test/tmp/substract.cpp
#include <iostream>
#include <stdio.h>
using namespace std;
int main( int argc, char *argv[], char *envp[] ) {
  if (argc < 3) {
    cout << "Invalid argument\n";
    cout << "Usage: substract <num1> <num2>\n";
  } else {
    int num1, num2, result;
    sscanf(argv[1], "%d", &num1);
    sscanf(argv[2], "%d", &num2);
    result = num1 - num2;
    cout << result << "\n";
  }
}
